api4/system.go

```diff
@@ -513,11 +513,6 @@ func pushNotificationAck(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if _, appErr := c.App.GetPostIfAuthorized(ack.PostId, c.AppContext.Session()); appErr != nil {
```

Comment:
It is not clear to me why we moved this inside the `ack.PostId != ""` condition. Can you clarify?

Reply:
While we clear channel & thread notifications, we do not pass PostId. While we are trying to ack those `clear notifications` .. they are failing. As we may or may not get the PostId from the client, I have moved this condition to post notification acknowledgement.